### DISCUSSION TITLE
creating a metal cluster input file in lanterns

### DISCUSSION THREAD

**Discussion by Nodirbek-pc** [community/community#3195] [Other Feature Feedback, Questions, & Ideas] [answered]
how to create an input file in the lammps program of a metal silver cluster. Is there a ready input file?

**Comment by evi-liu** [comment, marked as answer]
This space is currently reserved for feedback on Discussions and GitHub for Mobile. This is noted in our [README](https://github.com/github/feedback#github-public-feedback-discussions) and the [pinned discussion](https://github.com/github/feedback/discussions/1).

All other feedback at this time can be shared through our [support channel](https://support.github.com/contact/feedback) or our [community forum](https://github.community/).
